oceannavigator/frontend/src/components/Map.jsx

```diff
@@ -1330,11 +1335,17 @@ export default class Map extends React.PureComponent {
     if (this.scaleViewer != null) {
       this.map.removeControl(this.scaleViewer);
     }
+
+    let scale = currentScale;
+    if (Array.isArray(scale)) {
+      scale = scale.join(",");
+    }
```

Comment:
workaround for when the variable colour scale can be a string or an array...it should be an array at all times but fixing that entirely is for another time.